### PR TITLE
Tests for MLP Mixer

### PR DESCRIPTION
**Pull request by shireenchand**
Adding tests for MLP Mixer

### CODE REVIEW

**Comment by shireenchand** [comment]
Hi @juliagsy, can you please review this PR. I have added the test for the MLP Mixer Model!

**Comment by juliagsy** [comment]
Hey! Just to let you know I'm reviewing the PR already but am looking into a failure, if it's related to ivy or the pr itself, will get back to you soon! thanks!

File: ivy_models_tests/mlpmixer/test_mlpmixer.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    model.v = ivy.asarray(v)
+    logits = model(img, data_format=data_format)
+
+    # Cardinality test
+    assert logits.shape == tuple([ivy.to_scalar(batch_shape), num_classes])
+
+    # Value test
+    if load_weights:
+        np_out = ivy.to_numpy(logits)
+        true_indices = np.array([1, 6, 9, 7, 4])
```
Comment (juliagsy) [inline ivy_models_tests/mlpmixer/test_mlpmixer.py:67]: Hey! I've briefly helped out with some permutations updates. Could you please double check the indices and logits because the values are floating pretty much atm, thanks!

**Comment by shireenchand** [comment]
Hi @juliagsy, I have made the changes. Could you please review it 😄 
Thanks!

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hey! Thanks for the changes! I'm not sure if it's happening on your side, but I'm on latest docker and latest models, there seem to be an unidentical structure in weights (missing dropout attr), would you mind to upload the updated weights to hf again and test it out? (which means the tests are failing, models couldn't be loaded as well) thanks a lot!

**Comment by shireenchand** [comment]
Hi @juliagsy, I have updated the weights. Can you please review it
Thanks!

**Comment by juliagsy** [comment]
Thanks for the great work!
